lib/matplotlib/colorbar.py

```diff
@@ -1105,6 +1131,7 @@ def on_mappable_changed(self, mappable):
         by :func:`colorbar_factory` and should not be called manually.
 
         """
+        _log.debug('mappable changed')
```

Comment:
```suggestion
        _log.debug('colorbar mappable changed')
```
Giving slightly more context helps understanding the log output.